icu4c/source/tools/toolutil/pkg_genc.cpp

```diff
@@ -153,7 +153,12 @@ static const struct AssemblyType {
         "#endif\n"
         "\t.data\n"
         "\t.const\n"
+        /* macOS PPC should use .align instead .balign because is unknown pseudo ops for it*/
+        "#if defined(__APPLE__) && defined(__POWERPC__)\n"
+        "\t.align 16\n"
+        "#else\n"
         "\t.balign 16\n"
+        "#endif\n"
```

Comment:
Would it make sense to provide both versions at the same time, and just give them different names?

Reply:
Thanks, just pushed an updated version.